doc/SETUP.md

```diff
@@ -0,0 +1,100 @@
+# How to run Alby in the local development environment
+
+## 🚀 Quick Start
+
+### 💻 Load extension into browser
+
+- **Chrome**
+
+  - Go to the browser address bar and type `chrome://extensions`
+  - Check the `Developer Mode` button to enable it.
+  - Click on the `Load Unpacked Extension…` button.
+  - Select the extension’s dist directory: `dist/development/chrome`
+  - To see the debug console check the `inspect views` in the extension details
+
+- **Firefox**
+
+  - Load the Add-on via `about:debugging` => `This Firefox` as temporary Add-on. (`about:debugging#/runtime/this-firefox`)
+  - Choose a .xpi file or the `manifest.json` file in the extension's dist directory: `dist/development/firefox`
+  - [debugging details](https://extensionworkshop.com/documentation/develop/debugging/#debugging_popups)
+  - To see the debug console click "inspect" on the list of temporary extensions (`about:debugging#/runtime/this-firefox`)
+
+- **Opera**
+
+  - Load the extension via `opera:extensions`
+  - Check the `Developer Mode` and load as unpacked from extension’s extracted directory.
+
+To connect to a remote development LND node you can use a [test account](https://github.com/bumi/lightning-browser-extension/wiki/Test-setup)
+
+### Testnet/testing-accounts for development use localhost testnet
+
+For most people who are new to the btc lightning network, starting a test version of the lightning network environment locally is very helpful for developing wallets, so that they can transfer money with confidence.
+
+[Start the lightning network test environment locally and link to the Alby](https://github.com/getAlby/lightning-browser-extension/wiki/Start-the-lightning-network-test-environment-locally-and-link-to-the-Alby)
+
+## Project Structure
```

Comment:
Now that we have this here we do not need it in the main README anymore. Remove from README please.